src/modules/create-market/components/create-market-form-liquidity/create-market-form-liquidity.jsx

```diff
@@ -382,8 +378,15 @@ export default class CreateMarketLiquidity extends Component {
         singleOutcomeOrderBook: newMarket.orderBook[outcome] || {},
       }
       const action = augur.trading.simulateTrade(orderInfo)
-      if (createBigNumber(action.tokensDepleted, 10).lt(tickSize)) {
-        errors.price.push(`Est. Cost of trade must be at least ${tickSize}`)
+      if (createBigNumber(action.tokensDepleted, 10).lt(0.0001)) {
+        errors.price.push('Est. Cost of trade must be at least 0.0001')
```

Comment:
need to use tickSize, here is an example that exists in code.
`errors.price.push(`Price must be a multiple of ${tickSize}`)`

Reply:
Suggest removing this check, checking against tokensDepleted isn't needed.